content/en/docs/toolbox/auditing-operating.md

```diff
@@ -0,0 +1,385 @@
+ desc      | The description of the rule.
+ condition | A filtering expression that is applied against auditing log to check whether they matched the rule.
+ macro     | The conditions of the macro.
+ list      | Value of list.
+ alias     | Value of alias.
+ enable    | If false, the rule will not be effective.
+ output    | Specifies the message of alert.
+ priority  | The priority of the rule.
+
+ When an auditing log matched a rule in `archiving-rule` and the priority of rule is not less than `archivingPriority`, it will be stored for further use. When an auditing log matched a rule in `alerting-rule`, if the priority of the rule is less than `alertingPriority`, it will be stored for further use;  else it will generate an alert will be sent to the user. 
+
+#### Rule condition
+
+`Condition` is a filtering expression that can use comparison operators (=, !=, <, <=, >, >=, contains, in, like, regex) and can be combined using Boolean operators (and, or and not) and parentheses. Here are the supported Filters.
+
+ Filter | Description
+ ---    | --- 
+ Workspace                | The workspace which this audit event happened
+ Devops                   | The devops project which this audit event happened
+ Level                    | Level of auditing logs
+ RequestURI               | RequestURI is the request URI as sent by the client to a server.
+ Verb                     | Verb is the verb associated with the request
+ User.Username            | The name that uniquely identifies this user among all active users.
+ User.Groups              | The names of groups this user is a part of.
+ SourceIPs                | Source IPs, from where the request originated and intermediate proxies.
+ ObjectRef.Resource       | The resource of the object associated with the request.
+ ObjectRef.Namespace      | The namespace of the object associated with the request.
+ ObjectRef.Name           | The name of the object associated with the request.
+ ObjectRef.Subresource    | The subresource of the object associated with the request.
+ ResponseStatus.code      | Suggested HTTP return code for the request
+ ResponseStatus.Status    | Status of the operation.
+ RequestReceivedTimestamp | Time the request reached the apiserver.
+ StageTimestamp           | Time the request reached the current audit stage.
+
+ For example, to match all logs in the namespace `test`,
+
+```
+ObjectRef.Namespace = "test"
+```
+
+ to match all logs in the namespaces start with `test`,
+
+```
+ObjectRef.Namespace like "test*"
+```
+
+ to match all logs happened in the latest one hour,
+
+```
+RequestReceivedTimestamp >= "2020-06-12T09:23:28.359896Z" and RequestReceivedTimestamp <= "2020-06-12T10:23:28.359896Z"
+```
+
+#### Macro
+`Macro` is a rule condition snippets that can be re-used inside rules and even other macros. Macros provide a way to name common patterns and factor out redundancies in rules. Here¡¯s an example of a macro.
```

Comment:
->
A `macro` is a rule condition snippet that can be re-used inside rules and even other macros. Macros provide a way to name common patterns and factor out redundancies in rules. Here is an example of a macro.